index.html

```diff
@@ -9,63 +9,77 @@
+        <path d="M195.706 32.7109C193.965 32.7109 192.342 32.4709 190.837 31.991C189.362 31.4811 188.064 30.7312 186.942 29.7413C185.851 28.7514 184.98 27.5066 184.331 26.0067C183.712 24.5069 183.402 22.7522 183.402 20.7424C183.402 18.7627 183.712 17.0679 184.331 15.658C184.98 14.2182 185.821 13.0484 186.854 12.1485C187.887 11.2186 189.067 10.5437 190.395 10.1237C191.722 9.67379 193.08 9.44882 194.466 9.44882C196.03 9.44882 197.447 9.68879 198.715 10.1687C200.014 10.6487 201.12 11.3086 202.035 12.1485C202.979 12.9884 203.702 13.9933 204.203 15.1631C204.734 16.333 205 17.6078 205 18.9876C205 20.0075 204.72 20.7874 204.159 21.3273C203.598 21.8673 202.817 22.2122 201.813 22.3622L190.881 24.027C191.206 25.0169 191.87 25.7668 192.873 26.2767C193.876 26.7567 195.027 26.9966 196.325 26.9966C197.535 26.9966 198.671 26.8466 199.733 26.5467C200.825 26.2167 201.71 25.8418 202.389 25.4218C202.861 25.7218 203.259 26.1417 203.584 26.6817C203.908 27.2216 204.071 27.7915 204.071 28.3915C204.071 29.7413 203.451 30.7462 202.212 31.4061C201.268 31.916 200.205 32.261 199.025 32.4409C197.845 32.6209 196.738 32.7109 195.706 32.7109ZM194.466 15.0281C193.758 15.0281 193.139 15.1481 192.608 15.3881C192.106 15.628 191.693 15.943 191.368 16.333C191.044 16.6929 190.793 17.1129 190.616 17.5928C190.468 18.0427 190.38 18.5077 190.35 18.9876L197.919 17.7278C197.83 17.1279 197.506 16.5279 196.945 15.928C196.384 15.3281 195.558 15.0281 194.466 15.0281Z" fill="#3077C6"/> />
+      </symbol>
+      <symbol id="night" viewBox="0 0 30 30">
+        <svg width="30" height="30" viewBox="0 0 30 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+          <path d="M25.6101 10.6101V4.38992H19.3899L15 0L10.6101 4.38992H4.38992V10.6101L0 15L4.38992 19.3899V25.6101H10.6101L15 30L19.3899 25.6101H25.6101V19.3899L30 15L25.6101 10.6101ZM15 22.9576C13.8196 22.9576 12.6923 22.6923 11.6844 22.2281C14.4164 20.9682 16.3263 18.2096 16.3263 15C16.3263 11.7905 14.4164 9.03183 11.6844 7.77188C12.6923 7.30769 13.8196 7.04244 15 7.04244C19.3899 7.04244 22.9576 10.6101 22.9576 15C22.9576 19.3899 19.3899 22.9576 15 22.9576Z" fill="white"/>
+        </svg>
+      </symbol>
+      <symbol id="account" viewBox="0 0 40 40 >
+        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path d="M20 0C8.96 0 0 8.96 0 20C0 31.04 8.96 40 20 40C31.04 40 40 31.04 40 20C40 8.96 31.04 0 20 0ZM20 6C23.32 6 26 8.68 26 12C26 15.32 23.32 18 20 18C16.68 18 14 15.32 14 12C14 8.68 16.68 6 20 6ZM20 34.4C15 34.4 10.58 31.84 8 27.96C8.06 23.98 16 21.8 20 21.8C23.98 21.8 31.94 23.98 32 27.96C29.42 31.84 25 34.4 20 34.4Z" fill="white"/>
+    </svg>
+    </symbol>
+    </svg>
+    <svg>
```

Comment:
![image](https://user-images.githubusercontent.com/79144475/227653243-50673a6e-cca2-45f7-bcb0-ae8fe322908d.png)

Reply:
Here is an example using svg in a link

Reply:
Just check the path of this icon itself